packages/twenty-server/src/engine/metadata-modules/relation-metadata/relation-metadata.service.ts

```diff
@@ -441,6 +442,26 @@ export class RelationMetadataService extends TypeOrmQueryService<RelationMetadat
       columnName,
     );
 
+    const deletedAtFieldMetadata = await this.fieldMetadataRepository.findOneBy(
+      {
+        objectMetadataId: relationMetadata.toObjectMetadataId,
+        name: 'deletedAt',
+      },
+    );
+
+    if (!isDefined(deletedAtFieldMetadata)) {
+      throw new RelationMetadataException(
```

Comment:
I'm not sure we want to throw here? Not having a deletedAt column could be optional in the future. Wdyt?